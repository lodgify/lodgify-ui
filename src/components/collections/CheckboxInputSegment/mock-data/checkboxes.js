export const checkboxes = [
  {
    label: 'Wifi',
  },
  {
    label: 'Air Conditioning',
  },
  {
    label: 'Kitchen',
  },
  {
    label: 'TV',
  },
  {
    label: 'Washing Machine',
  },
  {
    label: 'Dishwasher',
  },
  {
    label: 'Oven',
  },
  {
    label: 'Heating Available',
  },
  {
    label: 'Clothes Dryers',
  },
  {
    label: 'Microwave',
  },
  {
    label: 'Hot Tub',
  },
];
