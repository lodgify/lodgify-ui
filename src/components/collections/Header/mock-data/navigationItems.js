export const navigationItems = [
  {
    text: 'Home',
  },
  {
    text: 'All properties',
    href: '/all-properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  { text: 'Contact Us', href: '/' },
  { text: 'What to do', href: '/' },
];

export const largeNavigationItems = [
  { text: 'Home', href: '/' },
  {
    text: 'All properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  { text: 'Contact Us', href: '/' },
  {
    text:
      'What to do when you go out on a Sunday and then somebody attacks you with big floppy fish',
    href: '/',
  },
];
