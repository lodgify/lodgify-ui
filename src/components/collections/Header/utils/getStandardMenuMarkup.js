import React, { Fragment } from 'react';
import { Menu } from 'semantic-ui-react';
import { size } from 'lodash';

import { buildKeyFromStrings } from 'utils/build-key-from-strings';
import { Button } from 'elements/Button';
import { Submenu } from 'elements/Submenu';

import { getSubmenuPointing } from './getSubmenuPointing';
import { getLinkMarkup } from './getLinkMarkup';

/**
 * @param  {Object}   props
 * @param  {number}   props.activeNavigationItemIndex
 * @param  {Object[]} props.navigationItems
 * @param  {Object}   props.primaryCTA
 * @return {Object}
 */
export const getStandardMenuMarkup = ({
  /* eslint-disable react/prop-types */
  activeNavigationItemIndex,
  navigationItems,
  primaryCTA,
  /* eslint-enable react/prop-types */
}) => (
  <Fragment>
    {navigationItems.map(({ subItems, text, href }, index) =>
      size(subItems) > 0 ? (
        <Submenu
          isMenuItem
          isSimple
          isTriggerUnderlined={index === activeNavigationItemIndex}
          isTriggeredOnHover
          items={subItems}
          key={buildKeyFromStrings(text, index)}
          pointing={getSubmenuPointing(
            index,
            size(navigationItems),
            !!primaryCTA
          )}
        >
          {text}
        </Submenu>
      ) : (
        getLinkMarkup(text, href, index, activeNavigationItemIndex)
      )
    )}
    {primaryCTA && (
      <Menu.Item className="no-underline" link>
        <Button isRounded onClick={primaryCTA.onClick}>
          {primaryCTA.text}
        </Button>
      </Menu.Item>
    )}
  </Fragment>
);
