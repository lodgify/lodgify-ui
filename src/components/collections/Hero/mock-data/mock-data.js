export const navigationItems = [
  { text: 'Home', href: '/' },
  {
    text: 'All properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
];

export const backgroundImageUrl =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg';
