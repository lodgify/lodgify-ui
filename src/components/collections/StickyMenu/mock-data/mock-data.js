export const menuItems = [
  {
    text: 'availability',
    link: '#availability',
    active: true,
  },
  {
    text: 'rates',
    link: '#rates',
  },
  {
    text: 'reviews',
    link: '#reviews',
  },
  {
    text: 'description',
    link: '#description',
  },
];

export const styles = {
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center',
  marginTop: '500px',
  height: '400px',
  width: '100%',
  backgroundColor: '#fdb2b2',
};
