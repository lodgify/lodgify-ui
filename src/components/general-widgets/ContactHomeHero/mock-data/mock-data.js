export const navigationItems = [
  { text: 'Home', href: '/' },
  {
    text: 'All properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  {
    text: 'LOL',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
];

export const currencyOptions = [
  { text: 'EUR', value: 'EUR', label: 'Euro' },
  { text: 'USD', value: 'USD', label: 'US Dollar' },
  { text: 'JPY', value: 'JPY', label: 'Japanese Yen' },
];

export const languageOptions = [
  { text: 'English', value: 'en' },
  { text: 'German', value: 'de' },
  { text: 'Japanese', value: 'jp' },
  { text: 'Italian', value: 'it' },
];

export const backgroundImageUrl =
  'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg';
