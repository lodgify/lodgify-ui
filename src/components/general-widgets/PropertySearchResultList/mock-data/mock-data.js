export const propertySearchResults = [
  {
    bathsNumber: 2,
    bathsText: 'baths',
    bedroomsNumber: 2,
    bedroomsText: 'bedrooms',
    bedsNumber: 2,
    bedsText: 'beds',
    guestsNumber: 2,
    guestsText: 'guests',
    imageUrl:
      'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    pricePerPeriod: '$28000',
    periodText: 'per month',
    propertyAmenities: ['Pool', 'Wifi', 'Washer', 'Kitchen'],
    propertyName: 'The Cat House',
    propertyType: 'Bed and breakfast',
    propertyUrl: '/',
    ratingNumber: 4.7,
  },
  {
    bathsNumber: 1,
    bathsText: 'baths',
    bedroomsNumber: 2,
    bedroomsText: 'bedrooms',
    bedsNumber: 3,
    bedsText: 'beds',
    guestsNumber: 4,
    guestsText: 'guests',
    imageUrl:
      'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    pricePerPeriod: '$1280',
    propertyAmenities: ['Pool', 'Wifi', 'Washer', 'Kitchen'],
    propertyName: 'Space Hub A-675',
    propertyType: 'Space Station',
    propertyUrl: '/',
    ratingNumber: 4.7,
  },
];
