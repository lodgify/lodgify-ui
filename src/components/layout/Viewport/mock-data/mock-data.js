export const navigationItems = [
  {
    text: 'Home',
  },
  {
    text: 'All properties',
    href: '/all-properties',
    subItems: [
      {
        href: '/la-casa-viva',
        text: 'La Casa Viva',
      },
      {
        href: '/la-casa-muerta',
        text: 'La Casa Muerta',
      },
      {
        href: '/the-white-lodge',
        text: 'The White Lodge',
      },
      {
        href: '/the-black-lodge',
        text: 'The Black Lodge',
      },
    ],
  },
  { text: 'Contact Us', href: '/' },
  { text: 'What to do', href: '/' },
];

export const images = [
  {
    alternativeText: 'Two cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Two cats',
  },
  {
    alternativeText: 'Two more cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Two more cats',
  },
  {
    alternativeText: 'Much cats',
    url: 'https://li3.cdbcdn.com/oh/522a12d9-ab51-4635-94c1-42536f286e4d.jpg',
    title: 'Much cats',
  },
];

export const currencyOptions = [
  { text: 'EUR', value: 'EUR' },
  { text: 'USD', value: 'USD' },
  { text: 'JPY', value: 'JPY' },
];
export const languageOptions = [
  { text: 'English', value: 'en' },
  { text: 'German', value: 'de' },
  { text: 'Japanese', value: 'jp' },
  { text: 'Italian', value: 'it' },
];
export const footerNavigationItems = [
  { href: '/', text: 'Home' },
  { href: '/contact', text: 'Contact' },
  { href: '/pool', text: 'Pool' },
  { href: '/spa', text: 'Spa' },
  { href: '/tennis-court', text: 'Tennis Court' },
  { href: '/gym', text: 'Gym' },
];
export const socialMediaLinks = [
  { href: 'https://twitter.com/lodgify', iconName: 'twitter' },
  { href: 'https://instagram.com/lodgify', iconName: 'instagram' },
];
