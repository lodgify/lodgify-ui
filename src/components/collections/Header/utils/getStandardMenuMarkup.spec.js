import React from 'react';
import { mount } from 'enzyme';

import { navigationItems } from '../mock-data/navigationItems';

import { getStandardMenuMarkup } from './getStandardMenuMarkup';

const getMarkupAsRenderedComponent = extraProps =>
  mount(
    <div>
      {getStandardMenuMarkup({
        activeNavigationItemIndex: 1,
        navigationItems,
        ...extraProps,
      })}
    </div>
  );

describe('getStandardMenuMarkup', () => {
  describe('by default', () => {
    it('should render the right structure', () => {
      const actual = getMarkupAsRenderedComponent();

      expect(actual).toMatchSnapshot();
    });
  });

  describe('if `props.primaryCTA` is passed', () => {
    const primaryCTA = { onClick: Function.prototype, text: 'someText' };

    it('should render the right structure', () => {
      const actual = getMarkupAsRenderedComponent({ primaryCTA });

      expect(actual).toMatchSnapshot();
    });
  });
});
