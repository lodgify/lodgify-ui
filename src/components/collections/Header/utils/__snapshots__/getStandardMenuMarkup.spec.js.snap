// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getStandardMenuMarkup by default should render the right structure 1`] = `
<div>
  <MenuItem
    active={false}
    href="/"
    key="Home0"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </MenuItem>
  <Submenu
    isMenuItem={true}
    isSearchable={false}
    isSimple={true}
    isTriggerUnderlined={false}
    isTriggeredOnHover={true}
    items={
      Array [
        Object {
          "href": "/la-casa-viva",
          "text": "La Casa Viva",
        },
        Object {
          "href": "/la-casa-muerta",
          "text": "La Casa Muerta",
        },
        Object {
          "href": "/the-white-lodge",
          "text": "The White Lodge",
        },
        Object {
          "href": "/the-black-lodge",
          "text": "The Black Lodge",
        },
      ]
    }
    key="All properties1"
    name={null}
    noResultsText="No results"
    onChange={[Function]}
    pointing="top left"
    value={null}
    willOpenAbove={false}
  >
    <Dropdown
      additionLabel="Add "
      additionPosition="top"
      className=""
      closeOnBlur={true}
      closeOnEscape={true}
      deburr={false}
      icon={
        <Icon
          className={null}
          color={null}
          hasBorder={false}
          isButton={false}
          isCircular={false}
          isColorInverted={false}
          isDisabled={false}
          isLabelLeft={false}
          labelText={null}
          labelWeight={null}
          name="caret down"
          path={null}
          size="small"
        />
      }
      item={true}
      minCharacters={1}
      name={null}
      noResultsMessage="No results"
      onChange={[Function]}
      openOnFocus={true}
      options={
        Array [
          Object {
            "href": "/la-casa-viva",
            "key": "La Casa Viva",
            "text": "La Casa Viva",
          },
          Object {
            "href": "/la-casa-muerta",
            "key": "La Casa Muerta",
            "text": "La Casa Muerta",
          },
          Object {
            "href": "/the-white-lodge",
            "key": "The White Lodge",
            "text": "The White Lodge",
          },
          Object {
            "href": "/the-black-lodge",
            "key": "The Black Lodge",
            "text": "The Black Lodge",
          },
        ]
      }
      pointing="top left"
      renderLabel={[Function]}
      scrolling={true}
      search={false}
      searchInput="text"
      selectOnBlur={true}
      selectOnNavigation={true}
      simple={true}
      trigger="All properties"
      upward={false}
      value={null}
      wrapSelection={true}
    >
      <Ref
        innerRef={
          Object {
            "current": <div
              aria-expanded="false"
              class="ui item simple scrolling top left pointing dropdown"
              role="listbox"
              tabindex="0"
            >
              All properties
              <i
                class="icon small"
              >
                <svg
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20,8.5l-8,7-8-7Z"
                    fill="currentColor"
                  />
                </svg>
              </i>
              <div
                class="menu transition"
              >
                <a
                  aria-checked="false"
                  aria-selected="true"
                  class="selected item"
                  href="/la-casa-viva"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    La Casa Viva
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/la-casa-muerta"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    La Casa Muerta
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/the-white-lodge"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    The White Lodge
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/the-black-lodge"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    The Black Lodge
                  </span>
                </a>
              </div>
            </div>,
          }
        }
      >
        <RefFindNode
          innerRef={
            Object {
              "current": <div
                aria-expanded="false"
                class="ui item simple scrolling top left pointing dropdown"
                role="listbox"
                tabindex="0"
              >
                All properties
                <i
                  class="icon small"
                >
                  <svg
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20,8.5l-8,7-8-7Z"
                      fill="currentColor"
                    />
                  </svg>
                </i>
                <div
                  class="menu transition"
                >
                  <a
                    aria-checked="false"
                    aria-selected="true"
                    class="selected item"
                    href="/la-casa-viva"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      La Casa Viva
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/la-casa-muerta"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      La Casa Muerta
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/the-white-lodge"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      The White Lodge
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/the-black-lodge"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      The Black Lodge
                    </span>
                  </a>
                </div>
              </div>,
            }
          }
        >
          <div
            aria-expanded={false}
            className="ui item simple scrolling top left pointing dropdown"
            name={null}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onMouseDown={[Function]}
            role="listbox"
            tabIndex={0}
          >
            All properties
            <Icon
              className=""
              color={null}
              hasBorder={false}
              isButton={false}
              isCircular={false}
              isColorInverted={false}
              isDisabled={false}
              isLabelLeft={false}
              labelText={null}
              labelWeight={null}
              name="caret down"
              onClick={[Function]}
              path={null}
              size="small"
            >
              <i
                className="icon small"
                onClick={[Function]}
              >
                <svg
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20,8.5l-8,7-8-7Z"
                    fill="currentColor"
                  />
                </svg>
              </i>
            </Icon>
            <DropdownMenu>
              <div
                className="menu transition"
              >
                <DropdownItem
                  active={false}
                  href="/la-casa-viva"
                  key="La Casa Viva"
                  onClick={[Function]}
                  selected={true}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="La Casa Viva"
                >
                  <a
                    aria-checked={false}
                    aria-selected={true}
                    className="selected item"
                    href="/la-casa-viva"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      La Casa Viva
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/la-casa-muerta"
                  key="La Casa Muerta"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="La Casa Muerta"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/la-casa-muerta"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      La Casa Muerta
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/the-white-lodge"
                  key="The White Lodge"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="The White Lodge"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/the-white-lodge"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      The White Lodge
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/the-black-lodge"
                  key="The Black Lodge"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="The Black Lodge"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/the-black-lodge"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      The Black Lodge
                    </span>
                  </a>
                </DropdownItem>
              </div>
            </DropdownMenu>
          </div>
        </RefFindNode>
      </Ref>
    </Dropdown>
  </Submenu>
  <MenuItem
    active={false}
    href="/"
    key="Contact Us2"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      Contact Us
    </a>
  </MenuItem>
  <MenuItem
    active={false}
    href="/"
    key="What to do3"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      What to do
    </a>
  </MenuItem>
</div>
`;

exports[`getStandardMenuMarkup if \`props.primaryCTA\` is passed should render the right structure 1`] = `
<div>
  <MenuItem
    active={false}
    href="/"
    key="Home0"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </MenuItem>
  <Submenu
    isMenuItem={true}
    isSearchable={false}
    isSimple={true}
    isTriggerUnderlined={false}
    isTriggeredOnHover={true}
    items={
      Array [
        Object {
          "href": "/la-casa-viva",
          "text": "La Casa Viva",
        },
        Object {
          "href": "/la-casa-muerta",
          "text": "La Casa Muerta",
        },
        Object {
          "href": "/the-white-lodge",
          "text": "The White Lodge",
        },
        Object {
          "href": "/the-black-lodge",
          "text": "The Black Lodge",
        },
      ]
    }
    key="All properties1"
    name={null}
    noResultsText="No results"
    onChange={[Function]}
    pointing="top left"
    value={null}
    willOpenAbove={false}
  >
    <Dropdown
      additionLabel="Add "
      additionPosition="top"
      className=""
      closeOnBlur={true}
      closeOnEscape={true}
      deburr={false}
      icon={
        <Icon
          className={null}
          color={null}
          hasBorder={false}
          isButton={false}
          isCircular={false}
          isColorInverted={false}
          isDisabled={false}
          isLabelLeft={false}
          labelText={null}
          labelWeight={null}
          name="caret down"
          path={null}
          size="small"
        />
      }
      item={true}
      minCharacters={1}
      name={null}
      noResultsMessage="No results"
      onChange={[Function]}
      openOnFocus={true}
      options={
        Array [
          Object {
            "href": "/la-casa-viva",
            "key": "La Casa Viva",
            "text": "La Casa Viva",
          },
          Object {
            "href": "/la-casa-muerta",
            "key": "La Casa Muerta",
            "text": "La Casa Muerta",
          },
          Object {
            "href": "/the-white-lodge",
            "key": "The White Lodge",
            "text": "The White Lodge",
          },
          Object {
            "href": "/the-black-lodge",
            "key": "The Black Lodge",
            "text": "The Black Lodge",
          },
        ]
      }
      pointing="top left"
      renderLabel={[Function]}
      scrolling={true}
      search={false}
      searchInput="text"
      selectOnBlur={true}
      selectOnNavigation={true}
      simple={true}
      trigger="All properties"
      upward={false}
      value={null}
      wrapSelection={true}
    >
      <Ref
        innerRef={
          Object {
            "current": <div
              aria-expanded="false"
              class="ui item simple scrolling top left pointing dropdown"
              role="listbox"
              tabindex="0"
            >
              All properties
              <i
                class="icon small"
              >
                <svg
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20,8.5l-8,7-8-7Z"
                    fill="currentColor"
                  />
                </svg>
              </i>
              <div
                class="menu transition"
              >
                <a
                  aria-checked="false"
                  aria-selected="true"
                  class="selected item"
                  href="/la-casa-viva"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    La Casa Viva
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/la-casa-muerta"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    La Casa Muerta
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/the-white-lodge"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    The White Lodge
                  </span>
                </a>
                <a
                  aria-checked="false"
                  aria-selected="false"
                  class="item"
                  href="/the-black-lodge"
                  role="option"
                  style="pointer-events: all;"
                >
                  <span
                    class="text"
                  >
                    The Black Lodge
                  </span>
                </a>
              </div>
            </div>,
          }
        }
      >
        <RefFindNode
          innerRef={
            Object {
              "current": <div
                aria-expanded="false"
                class="ui item simple scrolling top left pointing dropdown"
                role="listbox"
                tabindex="0"
              >
                All properties
                <i
                  class="icon small"
                >
                  <svg
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20,8.5l-8,7-8-7Z"
                      fill="currentColor"
                    />
                  </svg>
                </i>
                <div
                  class="menu transition"
                >
                  <a
                    aria-checked="false"
                    aria-selected="true"
                    class="selected item"
                    href="/la-casa-viva"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      La Casa Viva
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/la-casa-muerta"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      La Casa Muerta
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/the-white-lodge"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      The White Lodge
                    </span>
                  </a>
                  <a
                    aria-checked="false"
                    aria-selected="false"
                    class="item"
                    href="/the-black-lodge"
                    role="option"
                    style="pointer-events: all;"
                  >
                    <span
                      class="text"
                    >
                      The Black Lodge
                    </span>
                  </a>
                </div>
              </div>,
            }
          }
        >
          <div
            aria-expanded={false}
            className="ui item simple scrolling top left pointing dropdown"
            name={null}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onMouseDown={[Function]}
            role="listbox"
            tabIndex={0}
          >
            All properties
            <Icon
              className=""
              color={null}
              hasBorder={false}
              isButton={false}
              isCircular={false}
              isColorInverted={false}
              isDisabled={false}
              isLabelLeft={false}
              labelText={null}
              labelWeight={null}
              name="caret down"
              onClick={[Function]}
              path={null}
              size="small"
            >
              <i
                className="icon small"
                onClick={[Function]}
              >
                <svg
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20,8.5l-8,7-8-7Z"
                    fill="currentColor"
                  />
                </svg>
              </i>
            </Icon>
            <DropdownMenu>
              <div
                className="menu transition"
              >
                <DropdownItem
                  active={false}
                  href="/la-casa-viva"
                  key="La Casa Viva"
                  onClick={[Function]}
                  selected={true}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="La Casa Viva"
                >
                  <a
                    aria-checked={false}
                    aria-selected={true}
                    className="selected item"
                    href="/la-casa-viva"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      La Casa Viva
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/la-casa-muerta"
                  key="La Casa Muerta"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="La Casa Muerta"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/la-casa-muerta"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      La Casa Muerta
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/the-white-lodge"
                  key="The White Lodge"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="The White Lodge"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/the-white-lodge"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      The White Lodge
                    </span>
                  </a>
                </DropdownItem>
                <DropdownItem
                  active={false}
                  href="/the-black-lodge"
                  key="The Black Lodge"
                  onClick={[Function]}
                  selected={false}
                  style={
                    Object {
                      "pointerEvents": "all",
                    }
                  }
                  text="The Black Lodge"
                >
                  <a
                    aria-checked={false}
                    aria-selected={false}
                    className="item"
                    href="/the-black-lodge"
                    onClick={[Function]}
                    role="option"
                    style={
                      Object {
                        "pointerEvents": "all",
                      }
                    }
                  >
                    <span
                      className="text"
                    >
                      The Black Lodge
                    </span>
                  </a>
                </DropdownItem>
              </div>
            </DropdownMenu>
          </div>
        </RefFindNode>
      </Ref>
    </Dropdown>
  </Submenu>
  <MenuItem
    active={false}
    href="/"
    key="Contact Us2"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      Contact Us
    </a>
  </MenuItem>
  <MenuItem
    active={false}
    href="/"
    key="What to do3"
    link={true}
  >
    <a
      className="link item"
      href="/"
      onClick={[Function]}
    >
      What to do
    </a>
  </MenuItem>
  <MenuItem
    className="no-underline"
    link={true}
  >
    <div
      className="link item no-underline"
      onClick={[Function]}
    >
      <Button
        hasShadow={false}
        icon={null}
        isCompact={false}
        isDisabled={false}
        isFluid={false}
        isFormSubmit={false}
        isLoading={false}
        isOutlined={false}
        isPositionedRight={false}
        isRounded={true}
        isSecondary={false}
        onClick={[Function]}
        size={null}
      >
        <Button
          as="button"
          basic={false}
          circular={true}
          className=""
          compact={false}
          disabled={false}
          floated="left"
          fluid={false}
          loading={false}
          onClick={[Function]}
          secondary={false}
          size={null}
          type="button"
        >
          <Ref
            innerRef={
              Object {
                "current": <button
                  class="ui circular left floated button"
                  type="button"
                >
                  someText
                </button>,
              }
            }
          >
            <RefFindNode
              innerRef={
                Object {
                  "current": <button
                    class="ui circular left floated button"
                    type="button"
                  >
                    someText
                  </button>,
                }
              }
            >
              <button
                className="ui circular left floated button"
                onClick={[Function]}
                type="button"
              >
                someText
              </button>
            </RefFindNode>
          </Ref>
        </Button>
      </Button>
    </div>
  </MenuItem>
</div>
`;
